dags/bqetl_fivetran_costs.py

```diff
@@ -0,0 +1,78 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskMarker
+from airflow.sensors.external_task import ExternalTaskSensor
+from airflow.utils.task_group import TaskGroup
+import datetime
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.gcp import bigquery_etl_query, gke_command
+
+docs = """
+### bqetl_fivetran_costs
+
+Built from bigquery-etl repo, [`dags/bqetl_fivetran_costs.py`](https://github.com/mozilla/bigquery-etl/blob/main/dags/bqetl_fivetran_costs.py)
+
+#### Description
+
+Fivetran Logs
+#### Owner
+
+lschiestl@mozilla.com
+"""
+
+
+default_args = {
+    "owner": "lschiestl@mozilla.com",
+    "start_date": datetime.datetime(2023, 1, 18, 0, 0),
+    "end_date": None,
+    "email": ["telemetry-alerts@mozilla.com", "lschiestl@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+tags = ["repo/bigquery-etl"]
+
+with DAG(
+    "bqetl_fivetran_costs",
+    default_args=default_args,
+    schedule_interval="0 2 * * *",
+    doc_md=docs,
+    tags=tags,
+) as dag:
+
+    fivetran_costs_derived__destination__v1 = bigquery_etl_query(
+        task_id="fivetran_costs_derived__destination__v1",
+        destination_table="destination_v1",
+        dataset_id="fivetran_costs_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="lschiestl@mozilla.com",
+        email=["lschiestl@mozilla.com", "telemetry-alerts@mozilla.com"],
+        date_partition_parameter="submission_date",
+        depends_on_past=False,
+    )
+
+    fivetran_costs_derived__incremental_mar__v1 = bigquery_etl_query(
+        task_id="fivetran_costs_derived__incremental_mar__v1",
+        destination_table="incremental_mar_v1",
+        dataset_id="fivetran_costs_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="lschiestl@mozilla.com",
+        email=["lschiestl@mozilla.com", "telemetry-alerts@mozilla.com"],
+        date_partition_parameter="submission_date",
+        depends_on_past=False,
+    )
+
+    fivetran_costs_derived__monthly_cost__v1 = bigquery_etl_query(
+        task_id="fivetran_costs_derived__monthly_cost__v1",
+        destination_table="monthly_cost_v1",
+        dataset_id="fivetran_costs_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="lschiestl@mozilla.com",
+        email=["lschiestl@mozilla.com", "telemetry-alerts@mozilla.com"],
+        date_partition_parameter="submission_date",
+        depends_on_past=False,
+    )
```

Comment:
the tasks don't get called at the end? how do I set that up?

Reply:
fixed itself by adding the waiting for the fivetran I think :) 
I'm not sure anymore by I thought tasks need to be called in order to run in airflow 
```python 
with DAG() as dag:
  task = Operator()

  task
```

Reply:
> I'm not sure anymore by I thought tasks need to be called in order to run in airflow

No, they just need to be added to a DAG, and being instantiated within a `with DAG(...)` block does that automatically.